src/joystick/windows/SDL_dinputjoystick.c

```diff
@@ -236,69 +234,101 @@ SetDIerror(const char *function, HRESULT code)
 }
 
 static SDL_bool
-SDL_IsXInputDevice(const TCHAR *name, const GUID* pGuidProductFromDirectInput)
+SDL_IsXInputDevice(Uint16 vendor_id, Uint16 product_id, const char* hidPath)
 {
-    UINT i;
+    SDL_GameControllerType type;
 
     if (!SDL_XINPUT_Enabled()) {
         return SDL_FALSE;
     }
 
-    if (SDL_tcsstr(name, TEXT(" XINPUT ")) != NULL) {
-        /* This is a duplicate interface for a controller that will show up with XInput,
-           e.g. Xbox One Elite Series 2 in Bluetooth mode.
-         */
```

Comment:
Have you verified that the Xbox One Elite Series 2 in Bluetooth mode works correctly with this change?

Reply:
I checked Bluetooth Xbox One Elite (first version) and Bluetooth LE connection with Xbox One Series controllers - they both contain "_IG" in their device path.

I don't have Xbox One Elite Series 2. I'll try to find one and test with it in near days.

Reply:
Here is Interface Path from `Xbox One Series Model 1914` (VendorID: 0x045e, ProductID: 0x0b13): 
`\\?\HID#{00001812-0000-1000-8000-00805f9b34fb}&Dev&VID_5e&PID_0b13&REV_0505&987a14b7e3a7&IG_00#9&55c66e1&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}`

Reply:
I confirmed that the Xbox Elite Series 2 controller in Bluetooth mode is properly detected as an XInput device with this change.